inference-engine/src/transformations/src/transformations/convert_quantize_dequantize.cpp

```diff
@@ -0,0 +1,139 @@
+
+    ngraph::graph_rewrite_callback callback = [=](pattern::Matcher& m) {
+        auto pattern_map = m.get_pattern_map();
+        auto data = pattern_map[data_pattern];
+        auto input_low = pattern_map[input_low_pattern];
+        auto input_high = pattern_map[input_high_pattern];
+        auto output_low = std::dynamic_pointer_cast<opset1::Constant>(pattern_map[output_low_pattern]);
+        auto output_high = std::dynamic_pointer_cast<opset1::Constant>(pattern_map[output_high_pattern]);
+        auto fq = std::dynamic_pointer_cast<opset1::FakeQuantize>(pattern_map[fq_pattern]);
+        auto zero_point = pattern_map[zero_point_pattern];
+        auto scale = pattern_map[scale_pattern];
+        auto convert1 = pattern_map[convert1_pattern];
+        auto convert2 = pattern_map[convert2_pattern];
+        auto sub = pattern_map[sub_pattern];
+        auto mul = pattern_map[mul_pattern];
+
+        size_t levels = fq->get_levels();
+        if (levels != 256)
+            return false;
+
+        float out_low_val;
+        if (!op::util::get_single_value(output_low, out_low_val))
+            return false;
+        float out_high_val;
+        if (!op::util::get_single_value(output_high, out_high_val))
+            return false;
+        const auto& type = convert1->get_element_type();
+        switch (type) {
+        case element::Type_t::i8:
+            if (out_low_val != -128 || out_high_val != 127)
+                return false;
+            break;
+        case element::Type_t::u8:
+            if (out_low_val != 0 || out_high_val != 255)
+                return false;
+            break;
+        default:
+            return false;
+        }
+
+        auto new_out_low = std::make_shared<ngraph::opset1::Multiply>(
+                std::make_shared<ngraph::opset1::Subtract>(output_low, zero_point), scale);
+        auto new_out_high = std::make_shared<ngraph::opset1::Multiply>(
+                std::make_shared<ngraph::opset1::Subtract>(output_high, zero_point), scale);
+
+        // check if new_out_low/high shapes are broadcastable to FQ's input
+        auto data_shape = data->get_output_partial_shape(0);
+        auto out_low_shape = new_out_low->get_output_partial_shape(0);
+        if (out_low_shape.rank().get_length() > data_shape.rank().get_length())
+            return false;
+        auto out_high_shape = new_out_high->get_output_partial_shape(0);
+        if (out_high_shape.rank().get_length() > data_shape.rank().get_length())
+            return false;
+
+        auto fake_q = std::make_shared<ngraph::opset1::FakeQuantize>(data, input_low, input_high, new_out_low, new_out_high, levels);
+        fake_q->set_friendly_name(mul->get_friendly_name());
+
+        copy_runtime_info({data, input_low, input_high, output_low, output_high, fq, zero_point, scale, convert1, convert2, sub, mul},
+                          {data, input_low, input_high, new_out_low, new_out_high, fake_q});
```

Comment:
data, input_low, input_high, output_low, output_high, zero_point, scale are all not a part that you remove from the original graph, so only fq, convert1, convert2 should be replaced by fq. @GlebKazantaev, please correct me if I'm wrong.